### PR TITLE
kaizen: Publish wheels to GitHub Releases

### PR DESCRIPTION
**Pull request by mikeroll**
- Don't want to force the consumers to build this every time
- Don't want to commit to having a proper PyPI yet

https://github.com/alasco-tech/alasco-app/pull/18317/files

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -14,106 +9,35 @@ on:
...
-          python-version: "3.10"
-          architecture: ${{ matrix.target }}
-      - name: Build wheels
-        uses: PyO3/maturin-action@v1
-        with:
-          target: ${{ matrix.target }}
-          args: --release --out dist --find-interpreter
-          sccache: "true"
-      - name: Upload wheels
+      - name: Upload wheels # TODO
```
Comment (chrisittner) [inline .github/workflows/build.yml:39]: 🍿

**Review by lorinkoz — APPROVED** [review]
Do your thing!

**Comment by mikeroll** [comment]
@chrisittner @lorinkoz would you mac people mind testing this for me? https://github.com/alasco-tech/alasco-app/pull/18317/files

**Comment by lorinkoz** [comment]
> @chrisittner @lorinkoz would you mac people mind testing this for me? https://github.com/alasco-tech/alasco-app/pull/18317/files

Don't mind to test it, but if this is the endgame, i can only meh to that

**Comment by lorinkoz** [comment]
It worked for me!

**Comment by mikeroll** [comment]
@lorinkoz sure still meh - so not the endgame - but a bit less meh than source-only.

**Pull request merged**